.travis.yml

```diff
@@ -0,0 +1,39 @@
+sudo: required
+
+services:
+  - docker
+
+env:
+  global:
+  - secure: AlNK0cuvKINkOeojnT4cXRe3x0Y+zQpje+YVYH18+x7ksabNXzQ8mYSl3Z3sp2f1a/q3Ht5XDFRFPHseEy4g/KzHHTPfnuk3DXi2pGbpwPxu1+bOJJOvJioGBWJETEibBEi05SyhLmAGg0YR97FbetkbXyX8QLl0Y9HuYWYXc36RDk/AgPlLmjEBwik4icu+yZXskTddHoX7yDP8dxYk6GN3a5GMpE/M9QShCWSD5Vpy569koZxv3/HRNQLy623WtfLT+ZTxr/av1UfdOBb9nr4NT/JRzAiEeviWvB0rxXB3hGYdY5kfyQR46d9l79Mm4tx9Dukqn2lOm7Ud6FuwaubSqNk+XosOLhKYv1JZgcvhpVxdxN1p/joIdnPU4lKFVB+UKgZPKEr/XUYF2DF1Smoq5pcnLqfg4SqDCLV82s3d/y+mbQHFNigmWy6KkTrnr26tcnUYlrVxKNP1MxxLm/pUciNHcsRLIR9Y2aLMs6ydXOc2GuYRgS1Uy/cmuvLfdjGVlfLDp9ybPnybB/9P25++4hm3wJ8wMrB5a41LN/gJILBiLJ6B4OoBz3y7Erk/mEX6ZswNglKMUyMh7VqRVqucElt7w2YzwF0MNrmq+bxp+Y+MijyirYzW7e5BsTc3empqeoXCJHwuHGUqpGhCw5qh+uKNwMBYfA5JGh7rqgQ=
+  - secure: Czaur+blg/kwzPCuXLhvm73Tc5YZOlobMYC96iLghaVLlbjNiWl1EMPH78uAARBJT2EHpoXlmYS/dEWj5EKkcIIcRt1zwwKbB96LC1vDAwdGLPK2OlA1iZyCI7lRjt2on5hU5pegBRKwRme1hEb0ii1uYG6JL7FErs9fUa9NKY0RbV5/00ByT5Kzpd5yYD2LbihBgm9gEr8LySu3vCBaqzGZKgGJ55kpfWTG6ENcGojGLronmsdtUlqXCwZA29BvYX1h8yHfKMFKzh7Rsu3TTBlq81uB2ZgoGZgLGFIQ5jDKjdW6r4CktkZ/u9FMQpsa2RaVFTTR7VyzZoHjdXFjezSXkfIrQ3nzr+NMf9ks+v/o52BpO5GllQFirz5mu/AhlqgqsLrvToMamJpvqva35u4GvpxW1qGcuoeo8xduDoKJuQmaqsf9eELorlJIq+/tv4g60UIiI4Gul+/a/eXOo06GHJ3I0fuUotmabF48bZOdkseJRUuiH2UAGN8fSKY4sy4+8vNZP/hL1h+aixDyNTuHTGHLKtvIP4uet6bQOKtorJs4fBL/56ibWCdx3YzoJemaVzptkHUvQBl02crefhWY9ayv8MvaAf0JzvKaZT+VSStw7K2eNoyLAGvbuKKL1vmagGPDPlilpIY3k34nLSRtIKqcQhC9OhB0gxCacpM=
+  matrix:
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=2.2.16 ANSIBLE_VERSION=2.3.0.0
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=2.2.16 ANSIBLE_VERSION=2.4.1.0
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=2.2.16 ANSIBLE_VERSION=2.5.5
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=2.1.84 ANSIBLE_VERSION=2.3.0.0
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=2.1.84 ANSIBLE_VERSION=2.4.1.0
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=2.1.84 ANSIBLE_VERSION=2.5.5
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=0.9.9  ANSIBLE_VERSION=2.3.0.0
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=0.9.9  ANSIBLE_VERSION=2.4.1.0
+  - PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=0.9.9  ANSIBLE_VERSION=2.5.5
+
+script:
+  - docker build
+    --build-arg PACKER_VERSION=$PACKER_VERSION
+    --build-arg ANSIBLE_VERSION=$ANSIBLE_VERSION
+    -t solutiondrive/docker-aws-packer-ansible:packer$PACKER_VERSION-ansible$ANSIBLE_VERSION-awscli$AWSCLI_VERSION-inspec$INSPEC_VERSION .
```

Comment:
i would like it to tag the highest possible version from each application at latest (PACKER_VERSION=1.2.4 AWSCLI_VERSION=1.15.42 INSPEC_VERSION=2.2.16 ANSIBLE_VERSION=2.5.5 should be latest at the moment)

Reply:
Like it. Already thought about this.

Reply:
Do you think we should build some versions less? The matrix seems a bit to large for me, at least larger than really needed.

Reply:
I think we do not need to support ansible 2.3 and also inpsec 0.9

Reply:
Are you sure with ansible? Currently/lately we *only* support 2.3 in this container. To preserve fully compatibility with current infrastructure using this container I'd suggest to keep ansible 2.3.
With inspec I agree, I just was not sure if we really do not use 0.9.

Reply:
Yeah, ok! But would drop it as soon as possible